mapbox/dr-ui
# Create themes component to house shared themes for Tag and Note

Blocked by #216 

This PR adds a Theme "component",  using quotes here since it exports a JSON object, that stores shared theme data for `Tag` and `Note`. This PR also tests the color contrast between the background color and color in Themes to help make sure they are accessible.

To test:
1. npm ci
2. npm start
3. navigate to the /Themes page to see each note and, when available, coordinating tag.
